src/index.ts

```diff
@@ -0,0 +1,72 @@
+import { version, name } from "../package.json";
+import type { IBootStrapperArgs, ISettings } from "./types";
+
+import { ArgumentParser } from "argparse";
+import { existsSync, readFileSync } from "fs";
```

Comment:
Don't use synchronous methods, use fs/promises instead.


```suggestion
import { exists, readFile } from "fs/promises";
```